app/questionnaire/router.py

```diff
@@ -161,16 +174,24 @@ def get_first_incomplete_location_for_section(
         self, routing_path, section_id, list_item_id=None
     ):
         section_key = (section_id, list_item_id)
+        first_incomplete_block_id = routing_path[0]
```

Comment:
The `block_id` for `routing_path[0]` could be complete/in-progress. I understand why its set there but confused me the first time.